README.md

```diff
@@ -136,7 +143,7 @@ aind is expected to be used in conjunction with encryption of the client device,
 
 ## TODOs
 * Map different UID range per containers
-* Support remote connection from phones and tablets, ideally using Web browsers (noVNC?).
+* Support remote connection from phones and tablets, ideally using Web browsers (noVNC support).
```

Comment:
Is this already possible?

Reply:
Yes, already tested on my docker environment.